Base/usr/share/man/man1/stat.md

```diff
@@ -0,0 +1,36 @@
+## Name
+
+stat
+
+## Synopsis
+
+```**sh**
+$ stat [-L] file
+```
+
+## Description
+
+Display file status.
```

Comment:
This could be used as the summary and have a little bit more information here instead.